2
0.1
0.2
BasicUNet features: (6, 16, 32, 64, 128, 16).
Training ...
  0%|                                                     | 0/1 [00:00<?, ?it/s]Traceback (most recent call last):
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <module>
Traceback (most recent call last):
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<string>", line 1, in <module>
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 116, in spawn_main
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 116, in spawn_main
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 126, in _main
    exitcode = _main(fd, parent_sentinel)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 126, in _main
    exitcode = _main(fd, parent_sentinel)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 126, in _main
    self = reduction.pickle.load(from_parent)
    self = reduction.pickle.load(from_parent)
ModuleNotFoundError: No module named 'DosePredictionDataset'
ModuleNotFoundError: No module named 'DosePredictionDataset'
    self = reduction.pickle.load(from_parent)
ModuleNotFoundError: No module named 'DosePredictionDataset'
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 116, in spawn_main
    exitcode = _main(fd, parent_sentinel)
  File "/Users/wangyangwu/miniforge3/envs/maastro/lib/python3.9/multiprocessing/spawn.py", line 126, in _main
    self = reduction.pickle.load(from_parent)
ModuleNotFoundError: No module named 'DosePredictionDataset'
  0%|                                                     | 0/1 [00:00<?, ?it/s]